.github/workflows/pre-commit.yml

```diff
@@ -28,16 +23,23 @@ jobs:
       # on Pull Requests!).
       env:
         SKIP: end-of-file-fixer,trailing-whitespace,clang-format,eslint,no-commit-to-branch
+      with:
+        # Enable extra hooks that require special system dependencies that we
+        # cannot expect to be installed on all contributor's systems.
```

Comment:
Please add a note that the custom container image used for running this action contains those dependencies.